QuizApp/QuizApp/UI/Screens/Login/LoginViewModel.swift

```diff
@@ -41,15 +39,17 @@ class LoginViewModel: ObservableObject {
     }
 
     @MainActor
-    func pressedLoginButton() {
+    func pressedLoginButton(_ completion: @escaping (Bool) -> Void) {
```

Comment:
Why do we need completionHandler? Can't this VM have a reference to AppData and change it?